hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/context/properties/SerializableSemVersTest.java

```diff
@@ -65,7 +74,9 @@ void comparatorPrioritizesOrderAsExpected() {
         assertTrue(
                 SEM_VER_COMPARATOR.compare(semVerWith(1, 0, 1, "alpha.12345", null), semVerWith(1, 0, 1, null, "build"))
                         < 0);
-        assertTrue(SEM_VER_COMPARATOR.compare(semVerWith(1, 0, 1, null, "build"), semVerWith(1, 0, 1, null, null)) < 0);
+        assertTrue(SEM_VER_COMPARATOR.compare(semVerWith(1, 0, 1, null, "build"), semVerWith(1, 0, 1, null, null)) > 0);
+        assertTrue(SEM_VER_COMPARATOR.compare(semVerWith(1, 0, 1, null, null), semVerWith(1, 0, 1, null, null)) == 0);
+        assertTrue(SEM_VER_COMPARATOR.compare(semVerWith(1, 0, 1, null, "2"), semVerWith(1, 0, 1, null, "1")) > 0);
```

Comment:
Would you please add another test verify that a version with no build portion (config version) is less than a version with a build portion? This is the most likely scenario for a config-only update. Going from no build version to a build version.

Reply:
Added Kelly. Thanks